src/sagemaker_container_support/environment.py

```diff
@@ -0,0 +1,607 @@
+    """
+    return read_json(os.path.join(INPUT_CONFIG_PATH, RESOURCE_CONFIG_FILE))
+
+
+def read_input_data_config():  # type: () -> Dict[str, Dict[str, str]]
+    """Read the input data configuration from /opt/ml/input/config/inputdataconfig.json.
+
+        For more information about inpudataconfig.json:
+https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-dist-training
+
+    Returns:
+            input_data_config (dict[string, object]): contents from /opt/ml/input/config/inputdataconfig.json.
+
+                                For example, suppose that you specify three data channels (train, evaluation, and
+                                validation) in your request. This dictionary will contain:
+
+                                {'train': {
+                                    'ContentType':  'trainingContentType',
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                },
+                                'evaluation' : {
+                                    'ContentType': 'evalContentType',
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                },
+                                'validation': {
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                }}
+
+                                You can find more information about /opt/ml/input/config/inputdataconfig.json here:
+                                https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-inputdataconfig
+    """
+    return read_json(os.path.join(INPUT_CONFIG_PATH, INPUT_DATA_CONFIG_FILE))
+
+
+def channel_path(channel):  # type: (str) -> str
+    """ Returns the directory containing the channel data file(s) which is:
+    - <self.base_dir>/input/data/<channel>
+
+    For more information about channels: https://docs.aws.amazon.com/sagemaker/latest/dg/API_Channel.html
+
+    Returns:
+        (str) The input data directory for the specified channel.
+    """
+    return os.path.join(INPUT_DATA_PATH, channel)
+
+
+def gpu_count():  # type: () -> int
+    """The number of gpus available in the current container.
+
+    Returns:
+        (int): number of gpus available in the current container.
+    """
+    try:
+        cmd = shlex.split('nvidia-smi --list-gpus')
+        output = str(subprocess.check_output(cmd))
```

Comment:
returns "b'...'" in py3 (need to .decode('utf8'))

Reply:
I will add a test for it